thrifty-schema/src/main/java/com/microsoft/thrifty/schema/Constant.java

```diff
@@ -54,6 +63,10 @@ public ConstValueElement value() {
         return element.value();
     }
 
+    public Builder toBuilder(Constant constant) {
```

Comment:
The other new builders all share this same trait, so I'll just comment once here.

`toBuilder` here is an instance method on `Constant`, but it's taking another `Constant` as an argument.  Any reason we couldn't eliminate the formal parameter and just use `this` instead?  For example, this would be simpler:

```java
public Builder toBuilder() {
  return new Builder(element, namespaces(), type);
}
```